<!-- Mission de suivi de waypoints -->
<launch>
	<node name="cmd_ramped" pkg="actuators" type="cmd_to_ramp.py">
		<remap from="cmd_vel_ramped" to="robot/motion"/>
	</node>
	<node pkg="rviz_satellite" type="map2world_tf_broadcaster.py"
        name="broadcaster_fixed" />
	<node name="regulateur_champs" pkg="process" type="regulateur_champs.py">
		<remap from="gps/local_pose" to="robot/pose"/>
	</node>

	<node name="behavior_server" pkg="process" type="behavior_server.py">
		<remap from="gps/local_pose" to="robot/pose"/>
	</node>
	<node name="gui_behavior_adder" pkg="process" type="gui_behavior_adder.py"/>
</launch>
